src/components/BasicForm/FormItem.tsx

```diff
@@ -3,14 +3,15 @@ import style from './Formitem.module.scss'
 interface ItemProps {
   title: string
   type: string
-  placeholder: string
+  placeholder: string,
+  value?: string
```

Comment:
I think value should not be optional, as you'll need to provide a value when prefilling form data for `Edit` etc.